Assets/Scripts/Player/RevolverBehaviour.cs

```diff
@@ -8,27 +8,39 @@ public class RevolverBehaviour : MonoBehaviour
     private ComboBehaviour thisCombo;
     CharacterBehaviour thisPlayer;
 
-     public List<GameObject> bullets;
+    public List<GameObject> bullets;
     public List<GameObject> bulletPrefabs;
     public Transform bulletspawn,crouchingBulletSpawn;
     GameObject desiredBullet;
 
-    public bool hasTemporaryBullet;
+    public int hasTemporaryBullet;
```

Comment:
`has[...]` and `is[...]` variables are by convention booleans. It is a bit confusing to have this as an integer, and I would need to read the rest of the code to understand this variable now

Reply:
changed variable name to numberOfTemporaryBullets